src/components/registration/registration.jsx

```diff
@@ -0,0 +1,120 @@
+import React, { useState } from 'react';
+import './registration.css';
+import { useValid } from '@/hooks/use-valid';
+import CustomButton from '@/UI/customButton/cistomButton';
+import Tweeter from '@/icons/tweeter';
+import { useDispatch } from 'react-redux';
+import { setCurrentUser } from '@/slices/currentUserSlice';
+import { useNavigate } from 'react-router-dom';
+
+const Registration = () => {
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+  const [retryPassword, setRetryPassword] = useState('');
+  const checkPasswords = (password1, password2) => {
+    return password1 === password2;
+  };
+  const isPasswordValid = useValid(password, ['lengthcheck']);
+  const isEmailValid = useValid(email, ['isempty']);
+  const disabledState =
+    !checkPasswords(password, retryPassword) && isPasswordValid && isEmailValid;
+  const dispatch = useDispatch();
+  const navigate = useNavigate();
+  const HandleRegistration = (e) => {
+    e.preventDefault();
+    dispatch(
+      setCurrentUser({
+        userEmail: email,
+        userToken: '1',
+      })
+    );
+    navigate('/home');
+  };
+  return (
+    <section className='registration-section'>
+      <div className='registration-block'>
+        <div className='logo registration-block__logo'>
```

Comment:
Why is there an unused class here? It is about the 'logo'. And further everywhere